internal/reload.go

```diff
@@ -69,10 +75,29 @@ func Reload(ctx context.Context, doc *enigma.Doc, global *enigma.Global, silent
 }
 
 func logProgress(ctx context.Context, global *enigma.Global, reservedRequestID int, skipTransientLogs bool) {
+	InteractDef := enigma.InteractDef{
```

Comment:
Nit: Not sure how much of a performance sink this is but it seems like it might be an unnecessary allocation in case we are not in debug mode. Could be prevented by a rewrite so that the allocation is only done when needed, e.g. limit > 0?
(Truly nitpicking!)